template-for-repository/proofs/Makefile.common

```diff
@@ -271,6 +271,26 @@ PROOF_SOURCES ?=
 # report, making the proof run appear to "hang".
 CBMC_TIMEOUT ?= 21600
 
+# Proof writers could add function contracts or loop invariants in
+# CBMC proofs. They should follow three distinct processes:
+# 1. Check whether a function contract is sound, where CHECK_CONTRACTS
+#    contains the list of functions with contracts to be checked.
+# 2. Replace function calls with their correspondent function contracts,
+#    where REPLACE_WITH_CONTRACTS contains the list of functions with
+#    contracts to be replaced. One must check separately whether a function
+#    contract is sound before replacing it.
+# 3. Apply loop invariants during verification, where APPLY_LOOP_INVARIANTS
+#    contains the list of functions with loop invariants to be applied.
+#    This option is currently unavailable.
+CHECK_CONTRACTS ?= ""
+CHECK_CONTRACTS_SET := $(patsubst %,--enforce-contract %, $(CHECK_CONTRACTS))
```

Comment:
[Nitpick]

We do a very similar thing with `REMOVE_FUNCTION_BODY` -> `CBMC_REMOVE_FUNCTION_BODY`.
Can we follow the same convention and use `CHECK_FUNCTION_CONTRACTS` -> `CBMC_CHECK_FUNCTION_CONTRACTS`? Same for replacing contracts and applying loop invariants.

It would be obvious in the remainder of the file that the `CBMC_` variables meant to be passed to CBMC and the other one is set by users.

Reply:
Implemented as suggested.